assembly/integer/safe/u128.ts

```diff
@@ -102,19 +101,19 @@ import { LOAD } from 'internal/arraybuffer';
 
     static fromBytesLE(array: u8[]): u128 {
       assert(array.length && (array.length & 15) == 0);
-      var buffer = <ArrayBuffer>array.buffer_;
+      var buffer = changetype<usize>(array.buffer);
       return new u128(
-        LOAD<u64>(buffer, 0),
-        LOAD<u64>(buffer, 1)
+        load<u64>(buffer, 0),
+        load<u64>(buffer, 1)
```

Comment:
I guess it should be:
```ts
load<u64>(buffer, 1 * sizeof<u64>())
```
Interesting how this pass tests

Reply:
It's because this method is never tested ;-) I'm adding a copy from the other u128 tests.

Reply:
ok, great!